types/react-resizable/index.d.ts

```diff
@@ -0,0 +1,64 @@
+// Type definitions for react-resizable 1.7
+// Project: https://github.com/STRML/react-resizable
+// Definitions by: Harry Brrundage <https://github.com/airhorns>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
```

Comment:
Is TS 2.8 required or would an earlier version suffice?

Reply:
I copied the TS version from `react/index.d.ts` after I got a bunch of errors in the tests concerning react without it! I think that makes sense though that types depending on the react types would depend on at least the same minimum TS version?

Reply:
Yes, if you depend on something that requires 2.8, then you require 2.8 as well.  Just checking.